platform/darwin/src/MGLRasterSource.h

```diff
@@ -104,6 +104,109 @@ MGL_EXPORT
  */
 - (instancetype)initWithIdentifier:(NSString *)identifier configurationURL:(NSURL *)configurationURL tileSize:(CGFloat)tileSize NS_DESIGNATED_INITIALIZER;
 
+/**
+ Returns a raster source initialized an identifier, tile URL templates, and
+ options.
+
+ After initializing and configuring the source, add it to a map view’s style
+ using the `-[MGLStyle addSource:]` method.
+
+ #### Tile URL templates
```

Comment:
Redeclaring `-initWithIdentifier:tileURLTemplates:options:` in the abstract superclass gave us a common place to put this substantial amount of documentation. It’s important for the developer to know that the identically or similarly named initializers in the subclasses behave similarly – or even to think of MGLRasterSource and MGLVectorSource as sharing the same initializers – even if they don’t share the same implementation in mbgl under the hood (but the developer doesn’t need to know that).

If we need to eliminate the superclass method, we should at least factor this documentation out into a jazzy guide and link to it from each identical method.